src/utils/index.ts

```diff
@@ -39,7 +39,7 @@ export function getErrorMessage(value: unknown): string {
     }
 
     if (isRouteErrorResponse(value)) {
-        return value.data;
+        return value.data.message;
```

Comment:
Prolly if (typeof foo === 'string') ... else if (typeof foobar === 'string') ... else String(value.data)

Because the type of data is `any`.